src/pages/Home.vue

```diff
@@ -1,32 +1,34 @@
 <template>
-  <div class="flex flex-wrap justify-center">
-    <div class="mt-5 mx-4 p-10 max-w-4xl">
-      <h1 class="font-bold text-5xl flex-1 mb-4">
-        Top Games {{ currentYear }}
-      </h1>
-      <div class="flex mb-4">
-        <input
-          type="search"
-          v-model="searchInput"
-          v-on:keyup.enter="fetchGame"
-          class="flex-1 text-black px-4 py-2 bg-gray-800 text-gray-300 rounded-sm"
-          placeholder="Enter your favorite game..."
-        />
-        <button
-          @click="fetchGame"
-          class="py-2 px-5 uppercase font-bold bg-green-400 rounded-sm ml-2"
-        >
-          Search
-        </button>
+  <section class="container mx-auto grid sm:px-8 sm:py-12 sm:gap-x-8 md:py-16">
```

Comment:
@albertmir the container looks a bit large to me at >1440px 
Could set a max-width to it? something like: `xl:max-w-4xl`

Reply:
Tried with `xl:max-w-4xl` but I prefer the `md:max-w-5xl`. _4xl_ was a little bit small.

Reply:
Looks cool!